scripts/deno/iframe-titles.ts

```diff
@@ -0,0 +1,187 @@
+import { array, object, string } from 'https://cdn.skypack.dev/zod@3.17?dts';
+import {
+	DOMParser,
+	Element,
+	Node,
+} from 'https://deno.land/x/deno_dom@v0.1.22-alpha/deno-dom-wasm.ts';
+import { octokit } from './github.ts';
+
+const searchParams = new URLSearchParams({
+	'api-key': 'test',
+	'query-fields': 'body',
+	q: 'iframe',
+	'show-fields': 'body',
+	'page-size': String(100),
```

Comment:
Out of curiosity, whats the difference between this and just using a string literal `'100'`?

Reply:
I prefer the syntax highlighting that way.

Reply:
this doesn't apply to the current code, but a potential gotcha that I always find a little surprising is that `new String(100)` returns something different to `String(100)`:

```js
typeof String(100) === 'string'  // true
typeof new String(100) === 'string'  // false

typeof new String(100) === 'object'  // true
```